### DISCUSSION TITLE
[Help]: How to completelly uninstall container and setting from Synology Docker

### DISCUSSION THREAD

**Discussion by DanLBC** [mbentley/docker-omada-controller#461] [Help] [answered]
### Controller Version

v5.14.26

### Describe Your Issue or Question

Why I am asking for? 
I installed Omada container in the past, and it worked perfect. 
After upgrade to new version it stopped every 2 min.
I deleted container and installed it again. 
I seems to work, but I am not able to login on any port.
[omada-controller.csv](https://github.com/user-attachments/files/16735441/omada-controller.csv)

### Expected Behavior

new installation

### Steps to Reproduce

new instaltion

### How You're Launching the Container

```plain
Synology container manager
```

### Container Logs

```plain
csv added
```

### MongoDB Logs

```plain
dont know where to write this command
```

**Comment by mbentley** [comment, marked as answer]
Duplicate of https://github.com/mbentley/docker-omada-controller/issues/418

There is a software bug in the 5.14 version of the controller and that’s on TP-Link. Use 5.13 for now.

**Reply by mbentley** [reply]
Otherwise, I don't know how the Synology manages containers to remove it as the subject is asking.  `docker stop...` and `docker rm...` are the typical commands used by docker though.
